spfraib/twitter
# Reorganize training

## What changed

- Reorganized the folder `code/8-training_binary` which now contains 3 folders (including a README for the first two): 
   - `bert_models` containing the training py file for the BERT-based model
   - `glove_cnn_models` containing the training py file for the GloVe+CNN models
   - `archive` where I stored all former files
- Reorganized the folder `jobs/training_binary` which now contains 2 folders:
  - `bert_models` containing the batch file to train BERT-based models on our 5 classes. All former batch files for BERT-based models were saved in `jobs/training_binary_bert_models/archive`
  - `glove_cnn_models` containing the batch file to train GloVe+CNN-based models on our 5 classes

## How to test:
- Look at both training files and batch files to make sure there are no mistakes. This includes:
  - `twitter/code/8-training_binary/bert_models/train_bert.py`
  - `twitter/code/8-training_binary/glove_cnn_models/train_glove_cnn.py`
  - `twitter/jobs/training_binary/bert_models/train_bert_model.sbatch`
  - `twitter/jobs/training_binary/glove_cnn_models/train_glove_cnn_model.sbatch`
- Read the two READMEs and make sure everything is clear. This includes:
  - `twitter/code/8-training_binary/glove_cnn_models/README.md`
  - `twitter/code/8-training_binary/bert_models/README.md`
